### DISCUSSION TITLE
【要望】アドベントカレンダーの通知に記事のタイトルもあると嬉しい

### DISCUSSION THREAD

**Discussion by makinzm** [increments/qiita-discussions#582] [要望・アイデア・不具合]
いつも運営ありがとうございます。楽しく利用させていただいております。
本タイトルについて説明させていただきます。

# 要望した理由

- https://qiita.com/notifications で表示されるアドベントカレンダーの通知を見てもタイトルが分からず、中を覗くまでタイトルが分からなくたくさんのアドベントカレンダーを購読することが困難となるため

# 提案

1. 著者の部分を削除して、デフォルトの設定でタイトルをつける。
  - タイトルをつけると通知が長くなってしまうことが懸念です。ただ、私の一個人の意見としては「誰が書いたか」はあまり重要視していないため、最低限の情報にし通知のタイトルが長くなりにくくできると思いました(アドベントカレンダーについても冗長である気がするため 🎄のアイコンだけにしてみました)。

| 現状 | 変更後 |
| -- | -- |
| `author`が「`calendarTitle`」アドベントカレンダーに`投稿`しました。 | 🎄「`calendarTitle`」に「`articleTitle`」が`追加`されました。 |

2. 設定項目で通知をカスタマイズできるようにする。 
  - アドベントカレンダーを購読している方の中には、巷のアドベントカレンダーと同様に中身を開くまで記事のタイトルや中身について不明であった方が嬉しい方がいる可能性があります。そのため、設定の欄に通知をカスタマイズできたらより良いのかなと思いました。

**Comment by tanaka-asami415** [comment]
@makinzm 
こちらこそいつもQiitaをご利用いただきありがとうございます！
リクエストとご提案をありがとうございます。

直近の開発優先度などから今年の対応はできなさそうなのですが、
来年のアドベントカレンダーの改善の際、参考にさせていただきます！🙇
